fdbserver/TLogServer.actor.cpp

```diff
@@ -3251,6 +3252,7 @@ ACTOR Future<Void> tLogStart(TLogData* self, InitializeTLogRequest req, Locality
 	return Void();
 }
 
+// what does this do?
```

Comment:
are you sure you want CI this comment?

Reply:
I don't want to check in this comment. It's just a draft PR.